ps_imageslider.php

```diff
@@ -621,7 +617,7 @@ function() {
         return $html;
     }
 
-    public function getNextPosition()
+    private function getNextPosition()
```

Comment:
Sorry but we cannot accept this change. If developers are using this function, you will break the shops.

Reply:
Okay I will let him public